ansible/group_vars/all/fluentd.yml

```diff
@@ -0,0 +1,10 @@
+---
+# code: language=ansible
+# Path: ansible/group_vars/all/fluentd.yml
+# This file contains the variables for the Fluentd service
+fluentd_data_dir_name: svc_fluentd_data
+fluentd_data_dir: "{{ docker_volumes_dir }}/{{ fluentd_data_dir_name }}"
+fluentd_docker_image_name: fluent/fluentd
+fluentd_docker_image_version: edge
```

Comment:
Add a comment to explain the choice of "edge" ?

Reply:
It's because fluentd have "edge" instead of "latest" but it's same.
I'll add a comment for that.